docs/blog/2020-01-12-faster-sites-with-optimized-media-assets/index.md

```diff
@@ -6,7 +6,7 @@ excerpt: "Introducing two new Gatsby-Cloudinary plugins for optimized media mana
 tags: ["media", "cloudinary", "performance", "image", "video"]
 ---
 
-Plugins are at the heart of developing software with Gatsby because they not only extend Gatsby’s robust features but also make available, data from multiple sources. I’m excited to introduce you to two Gatsby plugins: gatsby-source-cloudinary and gatsby-transformer-cloudinary.
+Plugins are at the heart of developing software with Gatsby because they not only extend Gatsby’s robust features but also make available, data from multiple sources. I’m excited to introduce you to two Gatsby plugins: [gatsby-source-cloudinary](/packages/gatsby-source-cloudinary/) and [gatsby-transformer-cloudinary](https://www.npmjs.com/package/gatsby-transformer-cloudinary).
```

Comment:
```suggestion
Plugins are at the heart of developing software with Gatsby because they not only extend Gatsby’s robust features but also make available, data from multiple sources. I’m excited to introduce you to two Gatsby plugins: [gatsby-source-cloudinary](/packages/gatsby-source-cloudinary/) and [gatsby-transformer-cloudinary](/packages/gatsby-transformer-cloudinary/).
```